provider/ec2/local_test.go

```diff
@@ -1622,6 +1622,9 @@ func (t *localServerSuite) TestNetworkInterfacesForMultipleInstances(c *gc.C) {
 		ids[i] = inst.Id()
 	}
 
+	// Sort instance list so we always get consistent results
+	sort.Slice(ids, func(l, r int) bool { return ids[l] < ids[r] })
```

Comment:
instance.Id is:
type Id string

so you can just use
sort.Strings([]string(ids))

Reply:
This will cause a compiler error because the slice type is `[]instance.Id` which while equivalent to `[]string` confuses the Go compiler.